hydra-test-utils/src/Test/Plutus/Validator.hs

```diff
@@ -76,14 +81,25 @@ evaluateScriptExecutionUnits validatorScript redeemer =
     Left e ->
       Left ("unexpected failure: " <> show e)
  where
+  result ::
+    Either
+      (TransactionValidityError UTxO.Era)
+      ( Map
+          ScriptWitnessIndex
+          ( Either
+              ScriptExecutionError
+              ExecutionUnits
+          )
+      )
   result =
-    evaluateTransactionExecutionUnits
-      cardanoEra
-      systemStart
-      (LedgerEpochInfo epochInfo)
-      (LedgerProtocolParameters pparams)
-      (UTxO.toApi utxo)
-      body
+    (fmap . fmap . fmap) snd $
+      evaluateTransactionExecutionUnits
```

Comment:
I'm surprised this needed changing and the other calls to `evaluateTransactionExecutionUnits` not?

`(fmap . fmap . fmap) snd $` could be anything and this is hard to read (judging from a github diff I'm looking at now). What is happening here?

Reply:
This signature changed and we discard the new information. https://github.com/IntersectMBO/cardano-api/pull/555/files

Reply:
Oh cool, having logs sounds useful 😊